sdk/lib/opentelemetry/sdk/trace/export/batch_span_processor.rb

```diff
@@ -173,7 +181,18 @@ def export_batch(batch, timeout: @exporter_timeout_seconds)
           end
 
           def report_result(result_code, batch)
-            OpenTelemetry.logger.error("Unable to export #{batch.size} spans") unless result_code == SUCCESS
+            if result_code == SUCCESS
+              @metrics_reporter.add_to_counter('otel.bsp.export.success')
+              @metrics_reporter.add_to_counter('otel.bsp.export.spans', increment: batch.size)
```

Comment:
very small nit, should we have `otel.bsp.export.spans and `otel.bsp.dropped_span` have the same namespace? so, either `otel.bsp.export.dropped_span` or `otel.bsp.spans`? non blocking imo

Reply:
Hmm, so initially I 👍 this suggestion, but upon reflection I don't think they should share a namespace. `otel.bsp.export.spans` is intended to communicate the number of spans successfully exported. `otel.bsp.dropped_spans` is a count of spans dropped, which might not be due to a failed export (we have 3 reasons at least for dropping spans: termination timeout, full buffer, failed export).

Reply:
I could see renaming `otel.bsp.export.spans` -> `otel.bsp.exported_spans`, so back to 👍 . I'll do that.